cli/stream_command.go

```diff
@@ -2224,6 +2312,10 @@ func (c *streamCmd) prepareConfig(_ *fisk.ParseContext, requireSize bool) api.St
 		}
 	}
 
+	if (c.repubSource != "" && c.repubDest == "") || (c.repubSource == "" && c.repubDest != "") || (c.repubHeadersOnly && (c.repubSource == "" || c.repubDest == "")) {
+		fisk.Fatalf("must specify both --republish-source and --republish-destination")
```

Comment:
we check headers here but dont mention it in the error so might need to handle the headers only case seperately with a more, specific, descriptive error for if its set

Reply:
I was wondering about that, added now